gazebo_plugins/src/gazebo_ros_camera.cpp

```diff
@@ -447,13 +447,13 @@ void GazeboRosCamera::Load(gazebo::sensors::SensorPtr _sensor, sdf::ElementPtr _
 
     // camera_ projection matrix (same as camera_ matrix due
```

Comment:
We're also interested in exposing the R matrix parameters above. I'm not sure what is the best way to set a matrix from SDF.

Reply:
It appears that in gazebo classic and gazebo matrices are set elementwise with a new sdf tag for each settable entry. 
Unless there's a better way we could just do this.

```suggestion
    camera_info_msg.r[0] = _sdf->Get<double>("r_0", 1.0).first;
    camera_info_msg.r[1] = _sdf->Get(double>("r_1", 0.0).first
    ... and so forth
```

Reply:
Maybe @adityapande-1995 or @scpeters can recommend an approach.

Reply:
I _think_ we could also do something like this:
`<rectification_matrix>1 0 0 0 1 0 0 0 1</rectification_matrix>`

And while we're at it make the other matricies use this syntax as well.

Reply:
> I _think_ we could also do something like this: `<rectification_matrix>1 0 0 0 1 0 0 0 1</rectification_matrix>`
> 
> And while we're at it make the other matricies use this syntax as well.

yes, I think this would be easier than specifying all the matrix parameters individually, as long as we document the order of parameters. I recommend this approach

Reply:
should we block this until we add the `rectification_matrix` or merge as is?

Reply:
I think it is reasonable to merge as-is and then we can add `rectification_matrix` and the other matricies in a new PR

Reply:
Here is the rectification_matrix PR: https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1423